src/GraphQL/ResolveFieldContext/IResolveFieldContext.cs

```diff
@@ -13,6 +13,9 @@ namespace GraphQL
 {
     /// <summary>
     /// Contains parameters pertaining to the currently executing <see cref="IFieldResolver"/>.
+    /// This object is only valid during the execution of the field; it is re-used once the field
```

Comment:
A warning about this should be written in the largest font in the readme. This is a very dangerous change.

Reply:
I completely agree, although I do want to point out that this concept is not foreign to ASP.Net users.  HttpContext is also pooled.

Reply:
I did prevent reuse of resolvers that throw exceptions and resolvers that return data loaders.  It is very easy to write data loader code that uses the context during data loader postponed execution (and it would be normal to do so).  And for exceptions, it seems like it would be possible for error handlers to store a copy of the context in logging code or whatever.  (I know my code does.)  So for an extra allocation in case of error, it is not reused.

Reply:
This also needs to be written in the readme.

Reply:
Yup